src/plugin/markers/index.js

```diff
@@ -274,7 +274,7 @@ export default class MarkersPlugin {
             labelEl.innerText = label;
             labelEl.setAttribute('title', tooltip);
             this.style(labelEl, {
-                "font-family": "monospace",
+                "font-family": "inherit",
```

Comment:
Isn't it the same as just removing this prop?

Reply:
`font-family` does inherit the parent value, @SishaarRao in your testing when you removed this property did it not do this?